src/main/scala/nl.knaw.dans.easy.bagstore/ItemId.scala

```diff
@@ -33,12 +33,26 @@ abstract class ItemId(val uuid: UUID) {
 }
 
 object ItemId {
+  // accept both upper and lower case? https://stackoverflow.com/questions/8258480/type-of-character-generated-by-uuid
+  val uuidRegex = "[a-zA-Z0-9]{8}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{12}"
```

Comment:
```suggestion
  val uuidRegex = "[a-zA-Z0-9]{8}(-[a-zA-Z0-9]{4}){3}-[a-zA-Z0-9]{12}"
```
Made this suggestion before, can't it be place in some shared value?

https://github.com/DANS-KNAW/dans-bag-lib/blob/master/src/main/scala/nl/knaw/dans/bag/v0/DansV0Bag.scala#L156